sdks/python/apache_beam/testing/synthetic_pipeline.py

```diff
@@ -238,13 +241,25 @@ def get_range_tracker(self, start_position, stop_position):
       tracker = range_trackers.UnsplittableRangeTracker(tracker)
     return tracker
 
+  def _gen_kv_pair(self, index):
```

Comment:
@pabloem In Java SDK there is position (somewhere else called `offset`). I assumed I can use index here in the same way, WDYT?

Reply:
that sounds good to me.